Mathlib/Order/Defs/PartialOrder.lean

```diff
@@ -104,6 +104,19 @@ def decidableLTOfDecidableLE [DecidableRel (α := α) (· ≤ ·)] : DecidableRe
       else isTrue <| lt_of_le_not_le hab hba
     else isFalse fun hab' => hab (le_of_lt hab')
 
+namespace Decidable
+
+variable [DecidableRel (α := α) (· ≤ ·)]
+
+lemma not_lt_iff_not_le_or_ge : ¬a < b ↔ ¬a ≤ b ∨ b ≤ a := by
+  rw [lt_iff_le_not_le, not_and_iff_or_not_not, not_not]
+
+end Decidable
+
+attribute [local instance] Classical.propDecidable
+
+lemma not_lt_iff_not_le_or_ge : ¬a < b ↔ ¬a ≤ b ∨ b ≤ a := Decidable.not_lt_iff_not_le_or_ge
```

Comment:
Is there a better name for this? I used _ge_ here as this can be rewritten as `a >= b`, but I don't believe that's the correct naming scheme. I wasn't able to find a good example to base the name off, though.

Reply:
I think it's fine using `ge` when it refers to the opposite order relation to one which is also included in the theorem statement.